### PR TITLE
feat: add phpinfo

### PR DESCRIPTION
**Pull request by kesselb**

### CODE REVIEW

**Review by MichaIng — APPROVED** [review]
Good idea. This makes it easier for admins to check PHP configs, and whether theirs are applied as expected. I am thinking about OPcache, but helpful for other things as well.

Approving only the idea here, as I did not find time to test it yet.

File: templates/settings-admin.php [hunk 1]
```diff
@@ -365,6 +365,9 @@ function FormatMegabytes(int $byte): string {
 							<?php p($l->t('Extensions:')); ?>
 							<em id="phpExtensions"><?php p($_['php']['extensions'] !== null ? implode(', ', $_['php']['extensions']) : $l->t('Unable to list extensions')); ?></em>
 						</p>
+						<p>
+							<a target="_blank" href="<?= $_['phpinfoUrl'] ?>">phpinfo</a>
```
Comment (szaimen) [inline templates/settings-admin.php:369]: can we make this a button somehow? Currently it is not good to discover imho

**Review by szaimen — APPROVED** [review]
Tested and seems to work, just one small question :)

File: templates/settings-admin.php (hunk 1 above)
Comment (szaimen) [reply templates/settings-admin.php:369]: Or maybe use the primary color for it?

File: README.md [hunk 2]
```diff
@@ -127,3 +127,7 @@ php occ serverinfo:update-storage-statistics -v --output=json_pretty
     "num_storages_other": 33
 }
 ```
+
+Show phpinfo
+
+``php occ config:app:set --value=yes serverinfo phpinfo``
```
Comment (szaimen) [inline README.md:133]: why are you making the feature opt-in instead opt-out?

Comment (szaimen) [reply README.md:133]: Maybe make it opt-out instead?

**Pull request merged**